components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -99,7 +99,7 @@ object WebExtensionSupport {
      */
     @Suppress("MaxLineLength", "LongParameterList")
     fun initialize(
-        engine: Engine,
+        manager: WebExtensionRuntime,
```

Comment:
nit: parameter should be called runtime now.

Reply:
Updated